### PR TITLE
feat: add 'wc' to allowed schemes

### PR DESCRIPTION
**Pull request by GarageInc**
Support for wallet connect links (connection via in-app browser with using wallet connect)

telegram-cloud-photo-size-2-5382337953629594976-y

### CODE REVIEW

File: TMessagesProj/src/main/java/org/telegram/ui/bots/BotWebViewContainer.java [hunk 1]
```diff
@@ -97,7 +97,7 @@ public abstract class BotWebViewContainer extends FrameLayout implements Notific
     private final static int REQUEST_CODE_WEB_VIEW_FILE = 3000, REQUEST_CODE_WEB_PERMISSION = 4000, REQUEST_CODE_QR_CAMERA_PERMISSION = 5000;
     private final static int DIALOG_SEQUENTIAL_COOLDOWN_TIME = 3000;
 
-    private final static List<String> WHITELISTED_SCHEMES = Arrays.asList("http", "https");
+    private final static List<String> WHITELISTED_SCHEMES = Arrays.asList("http", "https", "wc");
```
Comment (iGroza) [inline TMessagesProj/src/main/java/org/telegram/ui/bots/BotWebViewContainer.java:100]: Actually for web3 bot devs

**Review by iGroza — APPROVED** [review]

File: TMessagesProj/src/main/java/org/telegram/ui/bots/BotWebViewContainer.java (hunk 1 above)
Comment (GarageInc) [reply TMessagesProj/src/main/java/org/telegram/ui/bots/BotWebViewContainer.java:100]: @DrKLO , @Bubu , @alabiaga can you take a look please?

Comment (sp1r1don) [reply TMessagesProj/src/main/java/org/telegram/ui/bots/BotWebViewContainer.java:100]: Oh, really need this change. I think WS must have option

**Comment by dkaraush** [comment]
Allowed schemas are now up to the server. Please, create a suggestion on bugs.telegram.org, Thanks!

**Comment by elix1er** [comment]
bump

**Pull request closed without merging**